### DISCUSSION TITLE
想要入正了 怎么删掉入库操作

### DISCUSSION THREAD

**Discussion by weilovelili** [pjy612/SteamManifestCache#356] [Question / Answer]
就删某个游戏 其他不删 怎么操作啊

**Comment by pjy612** [comment]
[wiki 里面自己看](https://github.com/pjy612/SteamManifestCache/wiki/SteamTools#steamtools-18-%E4%B9%8B%E5%90%8E%E5%A6%82%E4%BD%95%E5%8F%96%E6%B6%88%E8%A7%A3%E9%94%81)
